lib/archethic/beacon_chain/summary_aggregate.ex

```diff
@@ -188,6 +199,62 @@ defmodule Archethic.BeaconChain.SummaryAggregate do
     end)
   end
 
+  defp aggregate_network_patches(network_patches, subset) do
+    sampling_nodes = P2PSampling.list_nodes_to_sample(subset)
+
+    network_patches
+    |> Enum.filter(&(length(&1) == length(sampling_nodes)))
+    |> Enum.zip()
+    |> Enum.map(fn network_patches ->
+      network_patches
+      |> Tuple.to_list()
+      |> Enum.dedup()
+      |> resolve_patches_conflicts()
+    end)
+    |> List.flatten()
+  end
+
+  defp resolve_patches_conflicts([patch]), do: patch
+
+  defp resolve_patches_conflicts(conflicts_patches) do
+    splitted_patches = Enum.map(conflicts_patches, &String.split(&1, "", trim: true))
+
+    # Aggregate the conflicts patch to get a final network patch
+    # We can use mean as the outliers will only impact the way a node
+    # fetch data. Because we don't if the truth is coming from the outliers
+    # or not, the mean will result in the smallest approximation.
+    latency_patch =
+      splitted_patches
+      |> Enum.map(fn d ->
+        d
+        |> Enum.take(2)
+        |> Enum.map(&String.to_integer(&1, 16))
+      end)
+      |> Enum.zip()
+      |> Enum.map(fn x ->
+        x
+        |> Tuple.to_list()
+        |> Utils.mean()
```

Comment:
I think here we should use a median instead of a mean.
Let's say patches are `["F__", "F__", "F__", "0__", "0__"]`
in fact, "0" and "F" are pretty close on the globe, but doing a mean of this will result in (16+16+16+0+0) / 5 = 9.6 = "9"
But "9" is far from "0" or "F" while the result of the aggregation should be "F" or "0". So I guess a median would be better as it will return "F".

Reply:
Why not

Reply:
Why not

Reply:
But we should take care about the the edge case `["F__", "0__"]` a classical median will do the mean of bot values are there is an even number of values. In this case the median will return "8" which is a wrong value.
But in other way `["2__", "4__"]` the median will return "3__" which is the expected value.
maybe using 0..16 value to represent digits is a wrong way because 0..F digits is are cycle value

Reply:
Algorithm are heuristic so they should not be perfect for edge case,. It's just a approximation thought at specific time which was the best approach for normal cases. Of course we can improve later, but I would prefer to ship and see, otherwise it will always be a theory in our mind.

Also for the case when there are only two, I think it's fine to have a approximation , as it might be in bootstrapping mode before 3 nodes views.  The next patch will probably make it  change again.